apps/barista/src/shared/page.service.ts

```diff
@@ -14,18 +14,36 @@
  * limitations under the License.
  */
 
-import { HttpClient } from '@angular/common/http';
+import { HttpClient, HttpErrorResponse } from '@angular/common/http';
 import { Injectable } from '@angular/core';
-import { AsyncSubject, Observable } from 'rxjs';
-import { switchMap } from 'rxjs/operators';
+import { AsyncSubject, Observable, of } from 'rxjs';
+import { switchMap, tap, catchError } from 'rxjs/operators';
 
 import { BaLocationService } from './location.service';
 
 import { environment } from '../environments/environment';
-import { BaSinglePageContent } from '@dynatrace/barista-components/barista-definitions';
+import {
+  BaSinglePageContent,
+  BaLayoutType,
+  BaErrorPageContent,
+} from '@dynatrace/barista-components/barista-definitions';
 
 const CONTENT_PATH_PREFIX = 'data/';
 
+const ERRORPAGE_404: BaErrorPageContent = {
+  title: 'Error 404',
+  layout: BaLayoutType.Error,
+  errormessage:
+    'Sorry, the page you tried to access does not exist. Are you using an outdated link?',
+};
+
+const ERRORPAGE: BaErrorPageContent = {
+  title: 'Oops!',
+  layout: BaLayoutType.Error,
+  errormessage:
+    "Sorry, an error has occured. Don't worry, we're working to fix the problem!",
```

Comment:
Should we change the error message or implement a way so we get notified that this problem has been encountered (How can we fix a problem we do not know about ;) )